connPFM/cli/caps.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+
+
+def _get_parser():
+    parser = argparse.ArgumentParser(
+        description="Calculate CAPs on activity-inducing signal calculated with connPFM."
+    )
+
+    reqoptions = parser.add_argument_group("Required options")
+    reqoptions.add_argument("-i", "--input", dest="data_path", required=True, help="Input file")
+    reqoptions.add_argument("-o", "--output", dest="output", required=True, help="Output file")
+    reqoptions.add_argument("-a", "--atlas", dest="atlas", required=True, help="Atlas file")
+    reqoptions.add_argument("-m", "--mask", dest="mask", required=True, help="ROI mask")
+
+    optoptions = parser.add_argument_group("Optional options")
+    optoptions.add_argument(
+        "-n",
+        "--nsurrogates",
+        dest="nsur",
+        type=int,
+        default=1000,
```

Comment:
I would make this 100
```suggestion
        default=100,
```